frontend/public/components/masthead-toolbar.jsx

```diff
@@ -242,9 +242,11 @@ class MastheadToolbar_ extends React.Component {
         <Toolbar>
           <ToolbarGroup className="pf-u-screen-reader pf-u-visible-on-md">
             {/* desktop -- (updates button) */}
-            <Firehose resources={resources}>
-              <UpdatesAvailableButton onClick={this._onClusterUpdatesAvailable} />
-            </Firehose>
+            { flags[FLAGS.CAN_GET_CLUSTER_VERSION] &&
```

Comment:
I think we can use the existing `CLUSTER_VERSION` flag and skip the SSAR check. It should be false if you don't have authority.

Reply:
Wait, we still need the SSAR check to make sure that the initial API request for the cluster version only happens if the user has permissions.

Reply:
Or do we not care about the 403 from the feature check, just the one from this websocket?

Reply:
If it fails, that's OK. It's not any more expensive than doing the SSAR and saves a request on load. And we still avoid the watch when it won't work.

It does mean a harmless error in the console log, but we see that for other things.

Reply:
I think the bug was more about the repeated errors (presumably because firehose keeps retrying)